src/Dropdown/test/DropdownMenuSpec.js

```diff
@@ -234,6 +234,25 @@ describe('<Dropdown.Menu>', () => {
     assert.isNotNull(instance.querySelector('.rs-dropdown-item-submenu'));
   });
 
+  it('Should highlight menu item when hover', () => {
+    const { getByTestId } = render(
+      <DropdownMenu>
+        <DropdownItem data-testid="item-1">1</DropdownItem>
+        <DropdownItem data-testid="item-2">2</DropdownItem>
+        <DropdownItem data-testid="item-3">3</DropdownItem>
+        <DropdownItem data-testid="item-4">4</DropdownItem>
+      </DropdownMenu>
+    );
+
+    const menuItem = getByTestId('item-1');
+
+    act(() => {
+      Simulate.mouseOver(menuItem);
+    });
+
+    expect(menuItem).to.have.attr('class', 'rs-dropdown-item rs-dropdown-item-focus');
```

Comment:
You may use `.to.have.class()` assertion so that you don't have to write down the whole class name string.